src/target/riscv/riscv-013.c

```diff
@@ -120,6 +120,7 @@ typedef enum slot {
 typedef struct {
 	struct list_head list;
 	int abs_chain_position;
+	uint32_t dbgbase;
```

Comment:
Can you add a comment describing what this is for?

Reply:
done